### DISCUSSION TITLE
Possible thing we can exploit in the webstore?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1124] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I was accessing the secure shell extension(which my school system blocked) and by spamming the add to chrome button area and where the extension is blocked thing shows up, it said "Extension is already installed."

We probably can't do much with this, but it might be useful.

**Comment by s0urce-c0de** [comment]
My guess is that it creates the directory for the extension when you click on it, but it deletes the directory if you don’t have a policy that allows it. It might check if an extension is installed by of the directory exists. So if you time a click between those times, it might tell you that.
